polars/polars-core/Cargo.toml

```diff
@@ -134,6 +134,7 @@ arrow = { package = "arrow2", git = "https://github.com/jorgecarleitao/arrow2",
 # arrow = { package = "arrow2", version = "0.7", default-features = false, features = ["compute"] }
 chrono = { version = "0.4", optional = true }
 comfy-table = { version = "4.0", optional = true }
+getrandom = { version = "0.2", features = ["js"] }
```

Comment:
This one dhould be added by downstream users right? We don't need it in polars itself. Especially the js target not.